distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Telemetry.enso

```diff
@@ -0,0 +1,31 @@
+private
+
+import Standard.Base.Data.Dictionary.Dictionary
+import Standard.Base.Nothing.Nothing
+import Standard.Base.Data.Text.Text
+import Standard.Base.Data.Vector.Vector
+
+polyglot java import org.slf4j.LoggerFactory
+
+
+## Schedules a single telemetry message for sending.
+
+   Arguments:
+   - logger_name: Name of the (telemetry) logger
+   - msg: Message. Must not contain colon.
+   - arg_dict: Mapping of argument names to their values.
+log logger_name:Text msg:Text arg_dict:Dictionary -> Nothing =
+    arg_str = arg_dict.fold_with_key [] \acc arg_name arg_value ->
+        txt = arg_name + "=" + arg_value.to_text
```

Comment:
What happens if `arg_value.to_text` contains `=` or ` `?

Wasn't the whole idea to have `{}` here and pass arguments as a separate array?

Reply:
We should also probably disallow invalid `arg_name` (containing `=` or ` ` etc.)

Reply:
I guess it's ok to lack checks like this for a prototype, but would be good to at least create a ticket to follow up as when more usage of the Appender grows it may be easier to get such mistakes and they may not be as easy to detect.